marketplace/optimizely/src/optimizely-client.js

```diff
@@ -17,7 +17,7 @@ export default class OptimizelyClient {
       }
     });
 
-    if (response.status === 200) {
+    if (response.ok) {
```

Comment:
This fixes a bug where Optimizely sometimes gives us 2XX statuses, not just 200.